Gemfile

```diff
@@ -71,6 +71,7 @@ group :development, :test do
   gem "guard-rspec", require: false
   gem "guard-rubocop"
   gem "pry"
+  gem "dotenv-rails"
```

Comment:
I mean, there are no `.env.sample` file nor instruction that the environment variables should be inside a `.env` file...